src/main/java/mate/controller/CreateCarController.java

```diff
@@ -0,0 +1,39 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/create")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/cars/add")
```

Read common mistakes.